src/itemstackmetadata.cpp

```diff
@@ -9,6 +9,22 @@
 #define DESERIALIZE_KV_DELIM_STR "\x02"
 #define DESERIALIZE_PAIR_DELIM_STR "\x03"
 
+#define TOOLCAP_KEY "__tool_capabilities"
```

Comment:
We have the special fields `(empty)`, `description` and some for the coloring feature, but none of them with them underscores. How should this field be named?